akka-docs/src/main/paradox/remoting-artery.md

```diff
@@ -48,7 +48,7 @@ to your `application.conf` file:
 ```
 akka {
   actor {
-    provider = cluster 
+    provider = remote 
```

Comment:
I'm not sure we should even mention this anywhere given that we think people should use cluster, even if it is in the remoting docs.

Reply:
Hrm... subterfuge, I could get behind that. Removing those snippets all together?